### DISCUSSION TITLE
Oh My Darling, Mantine

### DISCUSSION THREAD

**Discussion by thinkdj** [mantinedev/mantine#3254] [Kind words]
I've always been hunting for good UI libraries, and always zeroed in on whatever I felt was _the best_ after an extensive search. 
I mean, there are hundreds of UI libraries and design systems out there, but only a handful of them cross a certain quality and aesthetic threshold. 

2019: Discovered _Ant Design_ and was impressed with the range of components and the level of fine-tuning each component provided 
2020: Discovered _MUI_ and was impressed by how the banal, overused Material Design was made into something so good
2021: Discovered _Atlaskit_ and _Elastic UI_, was wowed by the level of detail and thought put into each component, but was apprehensive about the licensing
2022: Discovered _Mantine_. I think my search ended here. For a long time to come.

The last couple of weeks, I've been building a framework for my future projects with Mantine as the UI base, and I must say, it's a _well thought out library in terms of UI, UX and DX_. The whole shebang!

Kudos, and thank you @rtivital and every single person who contributed to this project, for giving us such an amazing library

**Comment by rtivital** [comment]
Thanks for kind words!
